apps/front-end/front-office/src/app/components/drag-and-drop/drag-and-drop.component.ts

```diff
@@ -1,8 +1,66 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import {
+  CdkDragDrop,
+  moveItemInArray,
+  transferArrayItem,
+} from '@angular/cdk/drag-drop';
+
+interface Action {
+  name: string;
+  order: number;
+}
 
 @Component({
   selector: 'webonjour-drag-and-drop',
   templateUrl: './drag-and-drop.component.html',
   styleUrls: ['./drag-and-drop.component.scss'],
 })
-export class DragAndDropComponent {}
+export class DragAndDropComponent {
+  actions: Action[] = [
+    { name: 'Action 1', order: 1 },
+    { name: 'Action 2', order: 2 },
+    { name: 'Action 3', order: 3 },
+    { name: 'Action 4', order: 4 },
+  ];
+  sortedActions: Action[] = [{ name: 'Action 5', order: 5 }];
+
+  // eslint-disable-next-line @angular-eslint/use-lifecycle-interface
```

Comment:
Why these warnings are disabled?

Reply:
Because I don't understand why without he don't recognize the Interface Action